app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -1,107 +1,72 @@
 
-  useEffect(() => {
-    fetchNotes();
-  }, []);
+  const { t } = useTranslation();
 
   const fetchNotes = async () => {
     try {
-      setLoading(true);
+      setIsLoading(true);
       const {
         data: { notes },
       } = await notesApi.fetch();
       setNotes(notes);
     } catch (error) {
       logger.error(error);
     } finally {
-      setLoading(false);
+      setIsLoading(false);
     }
   };
 
-  if (loading) {
+  useEffect(() => {
+    fetchNotes();
+  }, []);
+
+  if (isLoading) {
     return <PageLoader />;
   }
 
   return (
-    <Container>
-      <Header
-        title="Notes"
-        actionBlock={
-          <Button
-            icon="ri-add-line"
-            label="Add new note"
-            size="small"
-            onClick={() => setShowNewNotePane(true)}
-          />
-        }
-        searchProps={{
-          value: searchTerm,
-          onChange: e => setSearchTerm(e.target.value),
-        }}
-      />
-      {notes.length ? (
-        <>
-          <SubHeader
-            rightActionBlock={
-              <Button
-                disabled={!selectedNoteIds.length}
-                icon={Delete}
-                label="Delete"
-                size="small"
-                onClick={() => setShowDeleteAlert(true)}
-              />
-            }
-          />
-          <Table
-            fetchNotes={fetchNotes}
-            notes={notes}
-            selectedNoteIds={selectedNoteIds}
-            setSelectedNoteIds={setSelectedNoteIds}
-          />
-        </>
-      ) : (
-        <EmptyState
-          image={EmptyNotesListImage}
-          primaryAction={() => setShowNewNotePane(true)}
-          primaryActionLabel="Add new note"
-          subtitle="Add your notes to send customized emails to them."
-          title="Looks like you don't have any notes!"
-        />
-      )}
-      <NewNotePane
-        fetchNotes={fetchNotes}
-        setShowPane={setShowNewNotePane}
-        showPane={showNewNotePane}
+    <>
+      <MenuBar
+        mainBlocks={MAIN_BLOCKS}
+        segmentBlocks={SEGMENT_BLOCKS}
+        showMenu={isMenuOpen}
+        tagBlocks={TAG_BLOCKS}
+        title={t("common.notes")}
       />
-      {showDeleteAlert && (
-        <DeleteAlert
-          refetch={fetchNotes}
-          selectedNoteIds={selectedNoteIds}
-          setSelectedNoteIds={setSelectedNoteIds}
-          onClose={() => setShowDeleteAlert(false)}
+      <Container>
+        <Header
+          menuBarToggle={() => setIsMenuOpen(isMenuOpen => !isMenuOpen)}
+          title={t("notes.allNotes")}
+          actionBlock={
+            <Button icon={Plus} label={t("notes.addNote")} size="small" />
+          }
+          searchProps={{
+            placeholder: t("notes.searchNotes"),
+            value: searchTerm,
+            onChange: e => setSearchTerm(e.target.value),
+          }}
         />
-      )}
-    </Container>
+        <Collection notes={notes} />
```

Comment:
Did you meant to `List` the notes here? Please check whether `List` would be a better component name rather than `Collection`.